.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+use Nexus\CsConfig\Factory;
+use PhpCsFixer\Finder;
+use Utils\PHPCSFixer\CodeIgniter4;
+
+$finder = Finder::create()
+	->files()
+	->in([
+		__DIR__ . '/system',
+		__DIR__ . '/tests',
+		__DIR__ . '/utils',
```

Comment:
`app` as well?

Reply:
I intentionally left `app/` because when we reach the `header_comment` rule I don't want to add the headers there. So, I'm planning a separate configuration file for `app` similar to this one except that `header_comment` is specifically set to `false`.